pkg/apis/migration/v1alpha1/migtoken_types.go

```diff
@@ -55,6 +56,7 @@ func init() {
 // Function to determine if a user can *verb* on *resource*
 // If name is "" then it means all resources
 // If group is "*" then it means all API Groups
+// if namespace is "" then it means all cluster scoped resources
 func (r *MigToken) CanI(client k8sclient.Client, namespace, resource, group, verb, name string) (bool, error) {
```

Comment:
would it be a bad idea to change this order to `group, resource, namespace`? (our mind it use to gvk, namespace, name) kind of pattern would be easier to read IMO